zeek

```diff
@@ -40,19 +40,64 @@ IMAGE_NAME="activecm/zeek:${zeek_release:-3.0}"
 CONTAINER_NAME="zeek"
 
 # If the current user doesn't have docker permissions run with sudo
-SUDO=''
 if [ ! -w "/var/run/docker.sock" ]; then
-	SUDO="sudo --preserve-env "
+	shopt -s expand_aliases
+	alias docker='sudo --preserve-env docker'
 fi
 
-if $SUDO docker inspect "$CONTAINER_NAME" &>/dev/null; then
-	CONTAINER_RUNNING=`$SUDO docker inspect -f "{{ .State.Running }}" $CONTAINER_NAME 2>/dev/null`
-	RESTART_POLICY=`$SUDO docker inspect -f "{{ .HostConfig.RestartPolicy.Name }}" $CONTAINER_NAME 2>/dev/null`
+if docker inspect "$CONTAINER_NAME" &>/dev/null; then
+	CONTAINER_RUNNING=`docker inspect -f "{{ .State.Running }}" $CONTAINER_NAME 2>/dev/null`
+	RESTART_POLICY=`docker inspect -f "{{ .HostConfig.RestartPolicy.Name }}" $CONTAINER_NAME 2>/dev/null`
 else
 	CONTAINER_RUNNING="false"
 	RESTART_POLICY="always"
 fi
 
+# initilizes Zeek directories and config files on the host
+init_cfg() {
+	# create a temporary container to run commands
+	local container="zeek-init-$RANDOM"
+	docker run \
+		--detach \
+		--name $container \
+		-v "$HOST_ZEEK":"/zeek" \
+		--network host \
+		"$IMAGE_NAME" \
+		sh -c 'while sleep 1; do :; done'
+	# ensure the temporary container is removed
+	trap "docker rm --force $container >/dev/null 2>&1" EXIT
```

Comment:
This creates a temporary docker container that we can run commands in or copy files from. The `while sleep 1` loop lets us do `docker exec` to run commands in it. The `trap` ensures this docker container is killed and removed regardless of how/where this script exits.